src/Api/Tracking.php

```diff
@@ -121,4 +122,39 @@ private static function get_logins(array $params): array
 
         return $response;
     }
+
+    /**
+     * Get last logins filtered by days.
+     *
+     * @param array $params refers to request params
+     * @return array Get logins.
+    */
+    private static function get_content_created(array $params): array
+    {
+        global $wpdb;
+
+        $response = [];
+
+        $query = $wpdb->get_results(
+            $wpdb->prepare(
+                "SELECT count(ID) as total, post_type
+                FROM wp_posts
+                WHERE post_date >= NOW() - INTERVAL %d  DAY
+                AND post_type IN ('post', 'page', 'p4_action')
```

Comment:
Does the campaign post type also need to be included here?

Reply:
The current query is also counting auto-draft posts. @GP-Dan-Tovbein, could you please double-check?

Reply:
Campaigns probably not, since are in decline. 

But we could amend the query to only track published content.